src/apps/frontend/app.global.css

```diff
@@ -0,0 +1,14 @@
+* {
```

Comment:
We want to avoid this file

Reply:
The purpose of including this file is to manage the margin and padding of the body, a task that, in my opinion, BaseWeb cannot handle.

Reply:
Possibly yes, but this opens up and can often act as a reference that its ok to include raw css

Reply:
Added a note saying `AVOID ADDING MORE CSS HERE. UTILIZE BASEWEB STYLING INSTEAD` in this file but didn't removed this file because i didn't find a way to change margin padding of body through baseweb

Reply:
What happens if we don't remove it for time being?